database/oracle/sqlpluscmd.pm

```diff
@@ -0,0 +1,466 @@
+#
+# Copyright 2018 Centreon (http://www.centreon.com/)
+#
+# Centreon is a full-fledged industry-strength solution that meets
+# the needs in IT infrastructure and application monitoring for
+# service performance.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package database::oracle::sqlpluscmd;
+
+use strict;
+use warnings;
+use centreon::plugins::misc;
+use Digest::MD5 qw(md5_hex);
+use File::Temp qw(tempfile);
+use Data::Dumper;
+
+my $debug = 0;
+
+sub printd  {

```

Comment:
you should use the output_add function with the debug flag set instead of custom function please